src/hooks/protections/can_apply_data.cpp

```diff
@@ -1552,17 +1558,26 @@ namespace big
 						{
 							if (model_info->m_vehicle_type != eVehicleType::VEHICLE_TYPE_SUBMARINECAR)
 							{
-								notify::crash_blocked(sender, "submarine car");
+								notify::crash_blocked(sender, "submarine car (sync)");
 								return true;
 							}
 						}
 					}
-					// blocks too many legit syncs
-					if (!NETWORK::NETWORK_IS_ACTIVITY_SESSION()) //If we're in Freemode.
+					else if (veh_creation_model != std::nullopt) 
 					{
-						// let's not get legit players with legit syncs targetted by trigger happy users
-						// notify::crash_blocked(sender, "submarine car freemode");
-						return true;
+						// object hasn't been created yet, but we have the model hash from the creation node
+						if (auto model_info = model_info::get_vehicle_model(veh_creation_model.value()))
+						{
+							if (model_info->m_vehicle_type != eVehicleType::VEHICLE_TYPE_SUBMARINECAR)
+							{
+								notify::crash_blocked(sender, "submarine car (creation)");
+								return true;
+							}
+						}
+					}
+					else // should (probably) never reach here
```

Comment:
Isn't this the scenario @Yimura was running into last night? Shouldn't we just ignore this?